bgpd/bgpd.c

```diff
@@ -5557,16 +5559,17 @@ int peer_password_unset(struct peer *peer)
 
 	/* Check if handling a regular peer. */
 	if (!CHECK_FLAG(peer->sflags, PEER_STATUS_GROUP)) {
+		/* Attempt to uninstall password on socket. */
```

Comment:
What doese this change accomplish?

Reply:
when unsetting the password on the bgp socket, first we should apply the password on the socket and then, reset the session or send update to peer.